### ISSUE TITLE
brotli still in supportedEncodings even when brotli option is not set.

### ISSUE THREAD

**Issue by JimmyHurrah** [fastify/fastify-compress#24]
If the accept-encoding header includes 'br' the code tries to use the brotli compressStream even when the brotli option is not present since it's still in the supportedEncodings array.

Req:
```accept-encoding: "br, gzip, deflate"```

Res:
```
Message: compressStream[encoding] is not a function
Object.Object.onSend in /app/node_modules/fastify-compress/index.js:110
Object.next in /app/node_modules/fastify/lib/hookRunner.js:47
Object.onSendHookRunner in /app/node_modules/fastify/lib/hookRunner.js:61
Object.onSendHook in /app/node_modules/fastify/lib/reply.js:140
_Reply._Reply.Reply.send in /app/node_modules/fastify/lib/reply.js:80
result.result.then in /app/node_modules/fastify/lib/handleRequest.js:92
unknown.[anonymous]:null
```

**Comment by mcollina** [comment]
Would you like to send a PR?

**Comment by delvedor** [comment]
Closed in #25
